pennylane_lightning/lightning_base.py

```diff
@@ -119,12 +135,30 @@ def capabilities(cls):
         # To be able to validate the adjoint method [_validate_adjoint_method(device)],
         #  the qnode requires the definition of:
         # ["_apply_operation", "_apply_unitary", "adjoint_jacobian"]
+        # pylint: disable=no-method-argument
         def _apply_operation():
             pass
 
+        # pylint: disable=no-method-argument, missing-function-docstring
         def _apply_unitary():
             pass
 
```

Comment:
Should this and others have a `self`?

Reply:
I think their name just need to exist and the functions return `None`, so `self` is not required. But we could add it for good measure.

Reply:
I think it's usually best practice to add the self when making base classes like this. Let's add them in